.github/workflows/ci_test.yml

```diff
@@ -52,3 +52,22 @@ jobs:
             echo export BAZEL_PATH=/d/a/addons/addons/bazel-%BAZEL_VERSION%-windows-x86_64.exe  >>D:\a\_temp\msys\msys64\home\runneradmin\.bash_profile
             echo export PATH=/c/hostedtoolcache/windows/Python/${{ matrix.python-version }}/x64/:$PATH  >>D:\a\_temp\msys\msys64\home\runneradmin\.bash_profile
             msys2do ./tools/ci_testing/addons_cpu.sh
+  sanity-cpu:
+    name: Python3 CPU sanity-check
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ['3.7.5']
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: CI sanity check py${{ matrix.python-version }}
+        run: |
+          set -x -e
+          python -m pip install wheel setuptools
+          wget --quiet -nc https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-darwin-x86_64.sh
+          chmod +x bazel-${BAZEL_VERSION}-installer-darwin-x86_64.sh
+          ./bazel-${BAZEL_VERSION}-installer-darwin-x86_64.sh --user
```

Comment:
Should be able to remove this. The shell script will install all necessary dependencies in the docker container

Reply:
it works now. summary:
- use `runs-on: ubuntu-18.04`
- adjust bazel for linux `...-installer-linux-x86_64.sh`
- call `bash tools/run_docker.sh -c 'make sanity-check'`

Reply:
Yes it runs, but those 4 lines are unnecessary and take time. The sanity check is ran inside of the docker container not on host. Please remove

Reply:
Oh yes right (－‸ლ)